monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -4470,6 +4470,23 @@ object Observable {
     override def empty[A]: Observable[A] =
       Observable.empty[A]
   }
+
+  /** [[cats.NonEmptyParallel]] instance for [[Observable]]. */
+  implicit def observableNonEmptyParallel: NonEmptyParallel[Observable, CombineObservable.Type] =
```

Comment:
Instances with no dependencies should be `val` and not `def`, otherwise it's going to create an unnecessary new instance every time it's required. Make this a `val` too.